app/assets/stylesheets/activities.css.scss

```diff
@@ -40,7 +40,7 @@ article {
 
 #scuba-diving-card{
           height: 176px;
-          background: url('scuba.jpg') center / cover;
+          background: image-url('scuba.jpg') center / cover;
```

Comment:
Line should be indented 2 spaces, but was indented 10 spaces<br>Prefer double-quoted strings